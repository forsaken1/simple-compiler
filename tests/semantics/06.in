int i;

void main() {
	i[0];
}