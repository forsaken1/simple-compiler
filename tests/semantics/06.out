Semantic error: Term not an array on line 4 in position 3